src/cli.js

```diff
@@ -17,6 +17,7 @@ const stripWarnings = require( './lib/strip-warnings' );
 
 module.exports = function( report, options ) {
 	const getESLintFormatter = format => {
+		// eslint-disable-next-line inclusive-language/use-inclusive-words
```

Comment:
Seems like we could change `See` to `Refer to` and start a new best practice

Reply:
It actually did not like 'master' in the URL, but I don't know what else I can do with it.

Reply:
Oh! I was going to suggest moving to a link that uses a specific SHA, but the link is broken! Because it didn't use a SHA-based link! 😂

So I guess remove it unless you can tell where it was pointing

Reply:
Oh, it is even broken 😲

Ok, I will take care of it once I get out of this traffic jam 🚦🚦🚦

Reply:
Fixed in 41903bf